src/modals/SettingsModal/SettingsModal.js

```diff
@@ -158,9 +158,42 @@ export default function SettingsModal({ show, hideModal }) {
           </Dropdown.Menu>
           <Form.Text className="text-muted mt-0">
             If the property or language is available, a second label will be
-            shown, useful when people is known with different names
+            shown, useful when people are known by different names
           </Form.Text>
         </Dropdown>
+        Right to nodes will be
```

Comment:
If you forgive me for the sentence above could you write this in english please 😂

Reply:
Sorry I wanted to code, but my mind was not in English